README.md

```diff
@@ -1,354 +1,50 @@
 
-Here is the home page for Cypress if someone wants to look it up
-<https://www.cypress.io/>
+### What is Dummygram?
 
-Cypress has very strong documentation so a new comer could find most of the information from their own site
-<https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell>
+- When you learn ReactJs, it seems obvious to work on some projects and this application is a clone of the original Instagram website and tends to have features inside it. So what are you waiting for? Use this opportunity to contribute and learn. :smiley:
 
-Also as a starting point it would be good to go through these tutorial videos
-<https://docs.cypress.io/examples/tutorials>
 
-</details>
-
----
-
-### Start Testing
-
-> You need to [setup](#project-setup) the project as usual, and after that we can run our test cases.
-
-<sub>See It Running</sub>
-
-<https://user-images.githubusercontent.com/84321236/223492442-1b161401-4ca2-41f7-832d-7302e2832b5a.mp4>
-
-<details><summary>See More</summary> <br>
-
-1. **Run This Command**
-
-```zsh
-npm run cypress:open
-```
-
-_It will open up this window,_
-
-<img src='https://i.ibb.co/QC72wq8/image.png' width='720'>
-
-> As you can see, we only added E2E testing
-
-2. **Click on E2E Testing & Select Browser**
-
-<img src='https://i.ibb.co/4TpgRgG/image.png' width='720'>
-
-3. **Now Click on any Test to Run it**
-
-<img src='https://i.ibb.co/z59yv8B/image.png' width='720'>
-
-</details>
-
----
-
-### Create New Test Cases
-
-Working Directory: `/cypress/` <br>
-E2E Files: `/cypress/e2e/`
-
-<img width="720" alt="image" src="https://user-images.githubusercontent.com/84321236/223501582-99163f05-940c-4e9a-a59a-8951a1be1e3a.png">
-
-<details><summary>See More</summary> <br>
-
-**To Create New E2E Tests**
-
-1. Goto `/cypress/e2e/`
-2. You can create new file (similar to `spec.cy.js`). <br>
-   **OR** add new `it` function inside existing `describe` function in this existing file.
-
-<img width="720" alt="test-file-screenshot" src="https://user-images.githubusercontent.com/84321236/223485219-2fd93fa7-686e-4a60-92b0-2a927ee4a7b0.png">
+<details><summary><h3>Open Source Programs this Repo had been a part of 😉  </h3></summary>
 
-3. Then, [test](#start-testing) your test cases:
-
-   ```zsh
-   npm run cypress:open
-   ```
+||
+|---|
+|<img width="300" src="https://user-images.githubusercontent.com/79099734/189589410-ca17afb8-5855-4316-918a-054f27594809.png">|
+|<img height="300" src="https://user-images.githubusercontent.com/77617189/205454287-c5bffb68-d08f-4103-a191-02b7f54fb81f.png">|
+|<img width="300" src="https://user-images.githubusercontent.com/77617189/205456062-b28a333d-c48b-41ef-96c3-676ef2363ade.png">|
+|<img width="300" src="https://user-images.githubusercontent.com/77617189/215256338-87d294c6-c33e-4da6-8236-f8ab6fae6c4b.png">|
+|<img width="300" src="https://user-images.githubusercontent.com/77617189/219851281-59de273a-388a-4367-a7c6-39d37196ee90.png">|
```

Comment:
img tag needs an alt

Reply:
@narayan954 They are still being displayed from the remote source.

Reply:
Well that doesn't excuse for an alt property